[2024-09-19T18:15:08.916+0000] {taskinstance.py:1147} INFO - Dependencies not met for <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-09-16T00:00:00+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0, success_setup=0, skipped_setup=0), upstream_task_ids={'start_task'}
[2024-09-19T18:15:08.962+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
[2024-09-19T18:23:35.210+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-09-19T18:23:35.234+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-09-19T18:23:35.236+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-19T18:23:35.296+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-09-16 00:00:00+00:00
[2024-09-19T18:23:35.325+0000] {standard_task_runner.py:57} INFO - Started process 222 to run task
[2024-09-19T18:23:35.369+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-09-16T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_generator.py', '--cfg-path', '/tmp/tmpysyoe489']
[2024-09-19T18:23:35.392+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask generate_customer_dim_data
[2024-09-19T18:23:35.864+0000] {task_command.py:415} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-09-16T00:00:00+00:00 [running]> on host 48913e9acee9
[2024-09-19T18:23:36.230+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='codewithyu' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-16T00:00:00+00:00'
[2024-09-19T18:23:36.257+0000] {logging_mixin.py:151} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2024-09-19T18:23:36.260+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-09-19T18:23:36.299+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, execution_date=20240916T000000, start_date=20240919T182335, end_date=20240919T182336
[2024-09-19T18:23:36.395+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-19T18:23:36.439+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
